database/seed.js

```diff
@@ -0,0 +1,40 @@
+const faker = require("faker");
+const MongoClient = require("mongodb").MongoClient;
+const assert = require("assert");
+const _ = require("lodash");
+const url = "mongodb://localhost:27017";
+
+const dbName = "comments";
+
+MongoClient.connect(url, function (err, client) {
+  assert.equal(null, err);
+
+  const db = client.db(dbName);
+  const commentsCollection = db.collection("comments");
+
+  let comments = [];
```

Comment:
I could be wrong, but it appears that there are multiple variables named 'comments'.  It's the name of the mongo collection (created in database/comments), and also seems to be an array (on line 15 of database/seed) that will be entered into the comments collection. You could add more clarity by Capitalizing the mongo collection name Comments (which I think may be considered best practice), and/or adding something descriptive to the array variable name (for instance comments on line 15 could be commentsArr - to make the distinction of an array of comments - not the actual collection).

Reply:
Adjusted per suggestion.